shuup/front/templates/shuup/front/macros/checkout.jinja

```diff
@@ -37,12 +37,12 @@
 {% macro address_phase_shipping_well(shipping_form, ship_to_billing_checkbox=True) %}
     <div class="well">
         <fieldset>
-            {{ render_address_form(_("Shipping address"), shipping_form, ship_to_billing_checkbox) }}
+            {{ render_address_form(_("Shipping address"), shipping_form, "shpping", ship_to_billing_checkbox) }}
```

Comment:
typo. Also if `render_address_form` is pre v0.5.1, we should add a changelog entry telling people about this new parameter.